packages/components/tooltip/src/tooltip.js

```diff
@@ -53,10 +71,14 @@ const Tooltip = (props) => {
     };
   }, []);
 
-  const { reference, popper } = usePopper({
+  const [referenceElement, setReferenceElement] = React.useState(null);
+  const [popperElement, setPopperElement] = React.useState(null);
+
+  const popper = usePopper(referenceElement, popperElement, {
     placement: props.placement,
-    modifiers: props.modifiers,
+    modifiers: [...(popperDefaultModifiers || props.modifiers)],
```

Comment:
Hmm I think we need to do a deep merge, as otherwise we might lose the default modifiers.

Reply:
Ah I did a mistake here.